content/knowledge-base/load-firebase-configuration.md

```diff
@@ -6,7 +6,7 @@ aliases:
   - '../custom-scripts/load-firebase-configuration'
 ---
 
-Instead of committing the Firebase configuration files to your repository, you can upload them to Codemagic as [environment variables](../building/environment-variables/) and reference them in a custom script. Note that the Firebase configuration file (`google-services.json` for Android or `GoogleService-Info.plist` for iOS) must be Base64-encoded.
+Instead of committing the Firebase configuration files to your repository, you can upload them to Codemagic as [environment variables](../building/environment-variables/) and reference them in a custom script. Note that the Firebase configuration file (`google-services.json` for Android or `GoogleService-Info.plist` for iOS) must be selected as **secure**.
```

Comment:
It's better to replace the path to environment variables with the new path (although the one here will probably also work). I would completely remove the last sentence here.